src/Content/Version.php

```diff
@@ -104,6 +104,23 @@ public function delete(): void
 		}
 	}
 
+	/**
+	 * Returns the changed fields, compared to the given version
+	 */
+	public function diff(VersionId|string $versionId, Language|string $language = 'default'): array
+	{
+		$versionId = VersionId::from($versionId);
+
+		if ($versionId->is($this->id) === true) {
+			return [];
```

Comment:
Unit test missing for this, but other than that I think it makes sense to have that method and implementation looks good.